sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -62,9 +62,10 @@ protected void ConfigureTestEventHub(IHostBuilder builder)
 
         protected (JobHost, IHost) BuildHost<T>(Action<IHostBuilder> configurationDelegate = null, Action<IHost> preStartCallback = null)
         {
+            var hostBuilder = new HostBuilder();
             configurationDelegate ??= ConfigureTestEventHub;
-
-            var hostBuilder = new HostBuilder()
+            configurationDelegate?.Invoke(hostBuilder);
```

Comment:
reverting

Reply:
though the delegate needs to be invoked before the rest of the configuration now. This may be related to the change to remove the ConfigureServices call.

Reply:
>though the delegate needs to be invoked before the rest of the configuration now. This may be related to the change to remove the ConfigureServices call.

There shouldn't be conflicting changes, why does it need to be invoked before?

Reply:
If it isn't invoked before, I get an error about not having the necessary sender/receiver.

Reply:
Nevermind, the issue was that it was after the build call.

Reply:
Actually.. It does need to before this call:
```cs 
.ConfigureDefaultTestHost<T>(b =>
                {
                    b.AddEventHubs(options =>
                    {
                        options.EventProcessorOptions.TrackLastEnqueuedEventProperties = true;
                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
                        options.CheckpointContainer = Guid.NewGuid().ToString("D").Substring(0, 13);
                    });
                })
```
Otherwise, when ConfigureInitialOffsetOptions is called, the options have not been bound yet.